cottoncandy/options.py

```diff
@@ -71,7 +71,7 @@ def get_config():
     return config
 
 cwd = os.path.split(os.path.abspath(__file__))[0]
-userdir = appdirs.user_data_dir("cottoncandy")
+userdir = appdirs.user_data_dir("cottoncandy",appauthor="cottoncandy")  # error on Windows 10, appdirs requires appauthor
```

Comment:
No need for the comment:
`userdir = appdirs.user_data_dir("cottoncandy",appauthor="cottoncandy") `